src/docs/firmware/battery.md

```diff
@@ -14,7 +14,7 @@ This method requires opening your device. Be sure to use the correct tools to pr
 2. Take out all of the screws.
 
 ::: tip
-Screws may be underneath "feet", or the material that keeps the Chromebook from sliding.
+Screws may be underneath "feet", or the material (usually rubber) that keeps the Chromebook from sliding.
```

Comment:
Wording on this one is weird in general

Reply:
What about 
"Screws may be underneath "feet", (usually made out of rubber) that keeps the Chromebook from sliding."

Reply:
Dont quote feet, that sounds even more weird. 

"Screws may be underneath the rubber feet that keeps the Chromebook from sliding."

never seen any not made of rubber that worked